index.js

```diff
@@ -30,7 +31,15 @@ Gulp.prototype.watch = function(glob, opt, task) {
     fn = this.parallel(task);
   }
 
-  return vfs.watch(glob, opt, fn);
+  var watcher = chokidar.watch(glob, opt);
+  if (fn) {
```

Comment:
Ok cool so this relates to your other point about bringing in docs and not just linking to them. 

You called out documenting the `close()` method so far. Do you want all the options described here or just the most typically used ones (like `usePolling` for network shares)? I'd recommend the latter - with a link to the rest.